lib/adapter.js

```diff
@@ -354,11 +362,21 @@ module.exports = (function () {
     })
   }
 
-  // This will convert and,or,like,in in where clause to $and,$or,$like,$in.
+  // This will convert and, or, like, in, >=, >, <=, < into $and, $or, $like, $in, $gte, $gt, $lte, $lt
   // Since the new format of sails v1 is not compatible with jsforce
   function parseClause(original) {
-    var replace = ['and', 'or', 'like', 'in']
-    const mapperFn = (value, key) => replace.indexOf(key) != -1 ? `$${key}`: key
+    const operatorMap = {
+      and: '$and',
+      or: '$or',
+      like: '$like',
+      in: '$in',
+      '>=': '$gte',
+      '>': '$gt',
+      '<=': '$lte',
+      '<': '$lt',
+    };
+
+    const mapperFn = (_value, key) => operatorMap[key] || key;
```

Comment:
Can you combine parseClause and parseDataByDatatype?
Looks like both parseClause and parseDataByDatatype recursively navigate into the object, can it be combined?

Reply:
Done. Also moved the function to a separate utils file